package.json

```diff
@@ -86,6 +86,7 @@
     "@types/node": "^20.14.9",
     "@types/semver": "^7.7.0",
     "@yarnpkg/types": "^4.0.1",
+    "es-iterator-helpers": "1.2.1",
```

Comment:
I wonder if we should move to Node 22 for dev instead?

Reply:
I'm unsure how straightforward it is to use one Node version for x and another for y. I'm curious to hear what @cometkim thinks!

Reply:
Oh, I see. How can we ensure that we don't use any newer Node features in our CLI scripts? That happened some time ago.

Reply:
We could run the test-integration step of the CI build with the oldest supported node version. (Not sure if that would catch everything though, it should also try `rescript format` then for example.)

Reply:
Yeah, that could work.